src/daily-auths-report.tsx

```diff
@@ -120,6 +120,38 @@ function tabulate(
   };
 }
 
+function tabulateSumByAgency(results: ProcessedResult[], filterIal?: number): TableData {
+  const filteredResults = results.filter((d) => !filterIal || d.ial === filterIal);
+
+  const days = Array.from(new Set(filteredResults.map((d) => d.date.valueOf())))
+    .sort((a, b) => a - b)
+    .map((d) => new Date(d));
+
+  const rolledup = rollup(
+    filteredResults,
+    (bin) => bin.reduce((sum, d) => sum + d.count, 0),
+    (d) => d.agency,
+    (d) => d.ial,
+    (d) => yearMonthDayFormat(d.date)
+  );
+
+  const header = ["Agency", "IAL", ...days.map(yearMonthDayFormat), "Total"];
+
+  const body = Array.from(rolledup)
+    .sort(([agencyA], [agencyB]) => ascending(agencyA, agencyB))
+    .flatMap(([agency, ials]) =>
+      Array.from(ials).map(([ial, ialDays]) => {
+        const dayCounts = days.map((date) => ialDays.get(yearMonthDayFormat(date)) || 0);
+
+        return [agency, String(ial), ...dayCounts, dayCounts.reduce((d, total) => d + total, 0)];
```

Comment:
Why does only the `ial` number need to be string-ified, and not the subsequent cells? Or does it?

Reply:
Numbers get right aligned and formatted with commas, and I figured IAL didn't need that treatment